heron/packing/src/java/com/twitter/heron/packing/utils/RamRequirement.java

```diff
@@ -0,0 +1,65 @@
+// Copyright 2016 Twitter. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+package com.twitter.heron.packing.utils;
```

Comment:
@billonahill The container and RamRequirements classes are only used by the packing algorithm. In that sense I don't see them as core components. In the context of packing I'm using them as helper classes that's why I put them under utils. If "utils" is misleading, then how do you think we should name this package?